### PR TITLE
[Jonathan Iheme] - Sage - Introduction to Zero-Knowledge Technology and its Blockchain Applications

### PR DESCRIPTION
**Pull request by 4undRaiser**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani & @Ishanpathak1

### CODE REVIEW

File: blog/02-9-2023-Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: Introduction to Zero-Knowledge Technology and it's Blockchain Applications
+description: Zero knowledge is a unique technology that helps push the boundary of what is possible in the web3 space. Building scalable trustless applications is vital for us to achieve a fully decentralized web. In this tutorial, I'll be introducing the technology and how it works and also it's different blockchain applications.
```
Comment (Ishanpathak1) [inline blog/02-9-2023-Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications.md:3]: Can you make the description of 18-20 words, as bigger than that it mess ups the website

File: src/data-tutorials/users.tsx [hunk 2]
```diff
@@ -845,6 +845,13 @@ const Users: User[] = [
     website: 'https://medium.com/celodevelopers/introduction-to-creating-nfts-on-celo-eb7240a71cc0',
     tags: ['beginner','foundation', 'nft', 'erc721'],
   },
+  {
+    title: "Introduction to Zero-Knowledge Technology and it's Blockchain Applications",
+    description: "Zero knowledge is a unique technology that helps push the boundary of what is possible in the web3 space. Building scalable trustless applications is vital for us to achieve a fully decentralized web. In this tutorial, I will be introducing the technology and how it works and also it's different blockchain applications.",
```
Comment (Ishanpathak1) [inline src/data-tutorials/users.tsx:850]: Change the description here too

**Review by Ishanpathak1 — COMMENTED** [review]
Kindly make the required changes

File: blog/02-9-2023-Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications.md (hunk 1 above)
Comment (4undRaiser) [reply blog/02-9-2023-Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications.md]: done

File: blog/02-9-2023-Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+---
+title: Introduction to Zero-Knowledge Technology and it's Blockchain Applications
+description: This tutorial is an introduction to zero-knowledge technology and some use cases in the blockchain space
+authors:
+  - name: ✍️ Jonathan Iheme
+url: https://github.com/4undRaiser
+image_url: https://avatars.githubusercontent.com/u/87926451?s=96&v=4
+tags: [celosage, intermediate]
+hide_table_of_contents: false
```
Comment (Ishanpathak1) [inline blog/02-9-2023-Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications.md:9]: set this to true

**Review by Ishanpathak1 — COMMENTED** [review]
Hey can you make the changes

**Review by viral-sangani — APPROVED** [review]

**Comment by viral-sangani** [comment]
@Ishanpathak1 @4undRaiser Cannot merge this right now since build is failing as you can see. I Please try running the docs locally, you will get an error, solve that error to get this merged.
Thanks

File: blog/02-9-2023-Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+title: Introduction to Zero-Knowledge Technology and it's Blockchain Applications
+description: This tutorial is an introduction to zero-knowledge technology and some use cases in the blockchain space
+authors:
+  - name: ✍️ Jonathan Iheme
+url: https://github.com/4undRaiser
+image_url: https://avatars.githubusercontent.com/u/87926451?s=96&v=4
+tags: [celosage, intermediate]
+hide_table_of_contents: true
+slug: "/tutorials/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications"
```
Comment (Ishanpathak1) [inline blog/02-9-2023-Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications.md:10]: Change it to small case

File: src/data-tutorials/users.tsx [hunk 5]
```diff
@@ -867,7 +867,14 @@ const Users: User[] = [
     tags: ['beginner','foundation', 'nft', 'erc721'],
   },
   {
-    title: ' How to Build Car Marketplace dapp Using React',
+    title: "Introduction to Zero-Knowledge Technology and it's Blockchain Applications",
+    description: "This tutorial is an introduction to zero-knowledge technology and some use cases in the blockchain space",
+    preview: require('./showcase/intermediate/introduction-to-zero-knowledge-technology-and-its-blockchain-applications.png'),
+    website: "blog/tutorials/Introduction-to-Zero-Knowledge-Technology-and-it's-Blockchain-Applications",
```
Comment (Ishanpathak1) [inline src/data-tutorials/users.tsx:873]: Change the casings here too.

**Review by Ishanpathak1 — COMMENTED** [review]
Make sure you make the changes and let me know after you make the changes

File: src/data-tutorials/users.tsx (hunk 5 above)
Comment (4undRaiser) [reply src/data-tutorials/users.tsx]: done

**Pull request closed without merging**